cmd/pktvisord/CrashpadHandler.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#define NOMINMAX
+
+#include "visor_config.h"
+#include <client/crash_report_database.h>
+#include <client/crashpad_client.h>
+#include <client/settings.h>
+
+namespace crashpad {
+
+static bool start_crashpad_handler(base::FilePath::StringType token, base::FilePath::StringType url, base::FilePath::StringType handler_path)
+{
+    std::map<std::string, std::string> annotations;
+    std::vector<std::string> arguments;
+    CrashpadClient client;
+    bool rc;
+
+    annotations["format"] = "minidump";
+    annotations["product"] = "pktvisor";
+    annotations["database"] = "pktvisor";
+    annotations["version"] = VISOR_VERSION_NUM;
+    annotations["token"] = token;
```

Comment:
is other information like OS, kernel or anything else sent automatically?

Reply:
Yes it does
```
uname.machine   amd64
uname.release   null
uname.sysname   Linux
uname.version   5.14.0 -1031-oem #34-Ubuntu SMP Fri Mar 25 08:21:05 UTC 2022 x86_64
....
```